### PR TITLE
docs: updating documentation

### LINKED ISSUE #158: Documentation review
This is a list of findings while I'm reviewing all the documentation files. The intention is to update it as I progress through the files.

| Done | File                                                             | Wording                                   | Issue/Suggestion                                      |
|------|------------------------------------------------------------------|-------------------------------------------|-------------------------------------------------------|
| [ ]  | [strd002.md](https://github.com/stride3d/stride-docs/blob/master/en/diagnostics/strd002.md) | N/A                                       | Same content as [strd001.md](https://github.com/stride3d/stride-docs/blob/master/en/diagnostics/strd001.md). They are stubs for when a currently open PR is merged |
| [ ]  | [strd003.md](https://github.com/stride3d/stride-docs/blob/master/en/diagnostics/strd003.md) | N/A                                       | Same content as [strd001.md](https://github.com/stride3d/stride-docs/blob/master/en/diagnostics/strd001.md). They are stubs for when a currently open PR is merged |
| [ ]  | [strd004.md](https://github.com/stride3d/stride-docs/blob/master/en/diagnostics/strd004.md) | N/A                                       | Same content as [strd001.md](https://github.com/stride3d/stride-docs/blob/master/en/diagnostics/strd001.md). They are stubs for when a currently open PR is merged |
| [ ]  | [strd007.md](https://github.com/stride3d/stride-docs/blob/master/en/diagnostics/strd007.md) | N/A                                       | Same content as [strd001.md](https://github.com/stride3d/stride-docs/blob/master/en/diagnostics/strd001.md). They are stubs for when a currently open PR is merged |
| [ ]  | [Stride.Utilities-remarks.md](https://github.com/stride3d/stride-docs/blob/master/en/examples/Stride.Utilities-remarks.md) | important in a Windows 8.x Store app | possible outdated information                           |
| [ ]  | [index.md](https://github.com/stride3d/stride-docs/blob/master/en/manual/index.md) | Follow [Stride on Twitter](https://twitter.com/stridedotnet?lang=en) | Follow [Stride on X](https://twitter.com/stridedotnet) |

### PR DESCRIPTION
**Pull request by CBID2**
## Description
This PR updates most of the documentation for Stride, per the review in issue #158.  

## Issue
Relates to #158 

## Note to Reviewers
This is still a work in progress 
**Update:** Ready for review

### CODE REVIEW

**Comment by CBID2** [comment]
@dotnet-policy-service agree

**Comment by CBID2** [comment]
Hi @Eideren! :) My PR is ready for your review! :)

File: en/manual/audio/non-spatialized-audio.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ To play non-spatialized audio at runtime, create an instance of it and define it
...
-| [Pan](xref:Stride.Audio.SoundInstance.Pan)       | Sets the balance between left and right speakers. By default, each speaker a value of 0.5. |
-| [Pitch](xref:Stride.Audio.SoundInstance.Pitch)     | Gets or sets the audio pitch (frequency). |
-| [PlayState](xref:Stride.Audio.SoundInstance.PlayState)	| Gets the state of the [SoundInstance](xref:Stride.Audio.SoundInstance). |
-| [Position](xref:Stride.Audio.SoundInstance.Position)	| Gets the current play position of the audio. |
-| [Volume](xref:Stride.Audio.SoundInstance.Volume)	| Sets the audio volume. |
+| Property                                               | Function                                                                                 |
+|--------------------------------------------------------|------------------------------------------------------------------------------------------|
+| [IsLooping](xref:Stride.Audio.SoundInstance.IsLooping) | Gets or sets looping of the audio.                                                       |
+| [Pan](xref:Stride.Audio.SoundInstance.Pan)             | Sets the balance between left and right speakers. By default, each speaker a value of 0. |
```
Comment (Eideren) [inline en/manual/audio/non-spatialized-audio.md:33]: Looks like your change for the sentence `each speaker a value of` wasn't included

**Review by Eideren — COMMENTED** [review]
Looks good, thanks, just one minor thing.
Could you also have a quick look at the changes to `toc.yml` @VaclavElias ? Looks fine to me, but just in case.

File: en/manual/audio/non-spatialized-audio.md (hunk 1 above)
Comment (CBID2) [reply en/manual/audio/non-spatialized-audio.md:33]: Ohh

Comment (CBID2) [reply en/manual/audio/non-spatialized-audio.md:33]: Done

**Comment by CBID2** [comment]
> Looks good, thanks, just one minor thing. Could you also have a quick look at the changes to `toc.yml` @VaclavElias ? Looks fine to me, but just in case.

Ohh, I hope I did the right changes @Eideren.

**Comment by VaclavElias** [comment]
I am reviewing this PR and testing locally..

There are 2 files which are referencing `entity-component-model` folder and need to be updated.

image

**Comment by VaclavElias** [comment]
Just a side note, there is also this section, which can be updated if you believe any your content updates need to be highlighted for a reader https://stride-doc-staging.azurewebsites.net/latest/en/manual/index.html#recent-updates. This makes sense mostly if there is something new/significant to be read.

**Comment by VaclavElias** [comment]
@Eideren, I tested `toc.yml` and other bits and all is working. 

Just 2 files mentioned above need to be fixed. Also, I discovered some unlinked pages, which are unrelated to this PR. I created a separate issue for that https://github.com/stride3d/stride-docs/issues/225.

Once those 2 files are updated with the correct folder name I can merge this PR.

**Comment by CBID2** [comment]
> entity-component-model

Hi @VaclavElias! :) I made those updates

File: en/manual/index.md [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
 
 ## Latest documentation
 
 ### Recent updates
 - <span class="badge text-bg-info">Updated</span> [Linux - Setup and requirements](platforms/linux/setup-and-requirements.md) - Fedora OS option added
 - <span class="badge text-bg-success">New</span> [Scripts - Serialization](scripts/serialization.md) - Serialization explained
 - <span class="badge text-bg-info">Updated</span> [Scripts - Public properties and fields](scripts/public-properties-and-fields.md) - Content improvements and additions
-- <span class="badge text-bg-success">New</span> [Engine - Entity Component model - Usage](engine/entity-component-model/usage.md) - ECS usage explained
+- <span class="badge text-bg-success">New</span> [Engine - Entity Component model - Usage](engine/entity-component-system/usage.md) - ECS usage explained
 - <span class="badge text-bg-info">Updated</span> [Engine - Entity Component model](engine/entity-component-model/index.md) - Content improvements
```
Comment (VaclavElias) [inline en/manual/index.md:17]: Here is the last escapee to be changed 🤣

Comment (VaclavElias) [reply en/manual/index.md:17]: ..Line 17

Comment (CBID2) [reply en/manual/index.md]: Now, I'm truly done @VaclavElias

Comment (CBID2) [reply en/manual/index.md]: > ..Line 17

Done and done! :)

**Pull request merged**